examples/load/README.md

```diff
@@ -0,0 +1,155 @@
+# File based task/template/handler definition
+
+This proposal introduces directory based task, template, and handler definition.
+
+## Configuration
+
+```
+[load]
+ enabled = true
+ dir="/path/to/directory"
+```
+
+`dir` specifies the directory where the definition files exist.
+
+The specified directory will check to see if any of three subdirectories `tasks`, `templates`, and `handlers` exists.
+
+The `tasks` directory will contain will contain task tickscripts and and the associated templated task
+definition files (either yaml or json).
+
+The `templates` directory will contain both template tickscripts
+
+The `handlers` directory will contain will contain all topic handler definitions in yaml or json.
+
+## Tasks
+### Tickscript (files with a `.tick` extension)
+
+Tickscripts can be marked up with commented key-value pairs for predefined attributes.
+Those attributes may be any of the following
+
+* `id` - the file name without the tick extension
+* `type` - determined by introspection of the task (stream, batch)
+* `dbrp` - defined using the `dbrp` keyword followed by a specified database and retention policy
+
+For example
+```
+dbrp "telegraf"."autogen"
+
+var measurement string
+var where_filter = lambda: TRUE
+var groups = [*]
+var field string
+var warn lambda
+var crit lambda
+var window = 5m
+var slack_channel = '#alerts'
+
+stream
+    |from()
+        .measurement(measurement)
+        .where(where_filter)
+        .groupBy(groups)
+    |window()
+        .period(window)
+        .every(window)
+    |mean(field)
+    |alert()
+         .warn(warn)
+         .crit(crit)
+         .slack()
+         .channel(slack_channel)
+```
+
+or
+
+```
+dbrp "telegraf"."autogen"
+
+stream
+    |from()
+        .measurement('cpu')
+        .groupBy(*)
+    |alert()
+        .warn(lambda: "usage_idle" < 20)
+        .crit(lambda: "usage_idle" < 10)
+        // Send alerts to the `cpu` topic
+        .topic('cpu')
+```
+
+### Templated Tasks
+
+Template variables may be added as either json or yaml.
```

Comment:
I think the README should have a simple TICKscript above to demonstrate creating tasks.

Then in this Template section it should first define a TICKscript template. Then show how that template can be referenced from a file in the `tasks` directory.

We want to make it clear that anything in the `tasks` dir will create a task, and anything in the `templates` dir will create a template.

Reply:
Added a section on defining a template before and the templated tasks now use that template. Think that  should be sufficient? Or did you have something else in mind?